tests/jballerina-unit-test/src/test/resources/test-src/expressions/stamp/map-stamp-expr-test.bal

```diff
@@ -206,6 +206,13 @@ function stampRecordTypeMultiDimensionMap() {
     map<Teacher> teacherMap = {"a": p1, "b": p2};
     map<map<Teacher>> multiMap = {"aa": teacherMap, "bb": teacherMap};
     map<map<Employee>> mapValue = checkpanic multiMap.cloneWithType(EmployeeDoubleMap);
+    assertMapValue_aa(mapValue);
+    assertMapValue_bb(mapValue);
```

Comment:
Shall we rename these two? I think we need to use camel case. https://learn-ballerina.github.io/best_practices/naming_conventions.html#functions
```suggestion
    assertMapValuePart1(mapValue);
    assertMapValuePart2(mapValue);
```